gnucash/report/standard-reports/transaction.scm

```diff
@@ -1513,7 +1521,22 @@ Credit Card, and Income accounts."))))))
            (string-append
             "<b>Transaction "
             (if (opt-val pagename-filter optname-transaction-matcher-regex) "regex" "substring")
-            " Matcher: </b>'" transaction-matcher "'<br>"))       
+            " Matcher: </b>'" transaction-matcher "'<br>"))
+       (string-append "<b>Void status: </b>" (symbol->string void-status) "<br>")
+       (if reconcile-status-filter
+	   (string-append
+	    "<b>Reconciled Status: </b>"
+	    (string-join
+	     (map (lambda (x) (assq-ref (list (cons #\c "cleared")
+					      (cons #\f "frozen")
+					      (cons #\n "non-reconciled/cleared")
+					      (cons #\y "reconciled")
+					      (cons #\v "void"))
+					x))
+			       reconcile-status-filter)
+	     ", ")
+	    "<br>")
+	   "")
```

Comment:
I mentioned before the infobox message have translation issues. I'm mentioning it again as this is in a separate commit which may or may not hide the old review comment.